### ISSUE TITLE
comunio.co.uk

### ISSUE THREAD

**Issue by smed79** [abp-filters/abp-filters-anti-cv#13]
**List the website(s) where you encounter issues with ABP filters**

```
http://www.comunio.de/?language=en_US
http://www.comunio.es/?language=en_US
http://www.comunio.it/?language=en_US
http://www.comunio.pt/?language=en_US
http://www.comunio.fr/?language=en_US
http://www.comunio.co.uk/?language=en_US
http://www.comunio.com.tr/?language=en_US
http://www.comunio-cl.com/?language=en_US
```

**Define the problem**

-  AdDefend / ad-reinsertion

**Ss**

-  https://i.imgur.com/dMrx4bm.png

**Tell us about your system configuration**

- Operating system and version: Manjaro 17
- Browser and version: Firefox 62 / Chromium 69
- Adblock Plus version: 3.3.2

**List all of your filter lists**

- ABP filters
- EasyList
- EasyPrivacy
- Fanboy's Annoyances
- Adblock Warning Removal List

**Other details**

- https://forums.lanik.us/viewtopic.php?p=141091#p141091
- https://git.io/fAd18

**Comment by mapx-** [comment]
well, this "special" list should deal with this crap using real filters not the known whitelisting ones.

When do you will use an `abort-on-property-read.js` - like snippet ?

**Comment by smed79** [comment]
`#@#` 220 and `@@` 339 filters after just 23 days of this [\_post\_](https://adblockplus.org/blog/newest-update-to-adblock-plus-is-a-massive-strike-against-anti-adblock-tools), is too much.

**Comment by KiLL4FuNK** [comment]
a question this is similar to something like ublock extra?

**Comment by KiLL4FuNK** [comment]
a question this is similar to something like ublock extra? 

@smed79 @mapx-

**Comment by mapx-** [comment]
Are you talking about this list ?

No, it's not about ublock extra (which is working already with ABP), but about (hypothetically) blocking sophisticated advertising (like all those ads blocked by uBo) using snippets (like scriptlets in uBo).

For the moment they are still working to have a collection of snippets and still use massively whitelisting filters (no difference with easylist)

**Comment by wizmak** [comment]
> ```
> http://www.comunio.de/?language=en_US
> http://www.comunio.es/?language=en_US
> http://www.comunio.it/?language=en_US
> http://www.comunio.pt/?language=en_US
> http://www.comunio.fr/?language=en_US
> http://www.comunio.co.uk/?language=en_US
> http://www.comunio.com.tr/?language=en_US
> http://www.comunio-cl.com/?language=en_US
> ```
@smed79 could you see ads on all of these domains? 
I'm only getting ads on comunio.de and comunio.es

**Comment by smed79** [comment]
Two months ago, [\_yes\_](https://github.com/uBlockOrigin/uAssets/commit/a5e963c0906749f866a935038b31741ee103cc7a#commitcomment-30604778) but not now. 
I see ads only on `comunio.es` and `comunio.de`

**Comment by wizmak** [comment]
Will be tackled with other websites with similar issue, closing this now.

**Comment by smed79** [comment]
@wizmak I see ads at `comunio.de`
